kythe/go/extractors/config/runextractor/README.md

```diff
@@ -32,6 +32,7 @@ Because `runextractor` is invoked in a manner not conducive to cleanly passing
 commandline flags, there is non-trivial setup done with environment variables.
 When calling `runextractor`, here are the relevant environment variables:
 
+* **KYTHE_CORPUS**: The corpus set for extracted files.
 * **KYTHE_ROOT_DIRECTORY**: The root path for file input to be extracted.
 * **KYTHE_OUTPUT_DIRECTORY**: The path for storing output.
 
```

Comment:
Include `REAL_JAVAC` and `JAVAC_EXTRACTOR_JAR` too?

Reply:
I was trying to ask whether those two environment variables should also be mentioned in this section. But if this is only the strictly-required ones, maybe not.